src/main/java/racingcar/RacingGameMachine.java

```diff
@@ -0,0 +1,26 @@
+package racingcar;
+
+public class RacingGameMachine {
+
+    private RacingGame racingGame;
+
+    public void play() {
+        preparation();
+        progress();
+        complete();
+    }
```

Comment:
RacingGameMaching에 있는 멤버메서드들이 RacingGame의 메서드들을 그저 실행시키고 있다는 생각이 듭니다.
다음과 같이 변경해도 좋을 것 같아요!
```suggestion
    public void play() {
        racingGame.preparation();
        racingGame.progress();
        racingGame.complete();
    }
```